pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -236,17 +240,19 @@ To be added as an admin to an existing project, run 'openshift ex policy add-use
 			o.Project = current.Name
 		}
 
-		if n := len(projects); n > 10 {
-			projects = projects[:10]
-			fmt.Printf("You have %d projects, displaying only the first 10. To view all your projects run 'osc get projects'.\n", n)
+		sort.StringSlice(projects).Sort()
+		fmt.Printf("\nYou have access to the following projects and can switch between them with 'osc project <projectname>':\n\n")
```

Comment:
If you already have a project set during login then you are probably not a first time user so don't dump the list of projects.